src/OrchardCore/OrchardCore.ContentManagement.Abstractions/Handlers/ValidateContentContext.cs

```diff
@@ -7,13 +13,27 @@ public ValidateContentContext(ContentItem contentItem) : base(contentItem)
         }
 
         public ContentValidateResult ContentValidateResult { get; } = new ContentValidateResult();
+
     }
 
     public static class ValidateContentContextExtensions
     {
-        public static void Fail(this ValidateContentContext context, params string[] errors)
+
+        public static void Fail(this ValidateContentContext context, params ValidationResult[] errors)
```

Comment:
NullReferenceException if the params is not set

Reply:
According to the doc

> If you send no arguments, the length of the params list is zero.